modules/apps/export-import/export-import-web/src/main/java/com/liferay/exportimport/web/internal/portlet/action/ExportImportMVCActionCommand.java

```diff
@@ -224,8 +228,16 @@ protected void importData(
 					ExportImportConfigurationConstants.TYPE_IMPORT_PORTLET,
 					importPortletSettingsMap);
 
-		_exportImportService.importPortletInfoInBackground(
-			exportImportConfiguration, inputStream);
+		long targetGroupId = MapUtil.getLong(
```

Comment:
Hey @GabrielHenriquedev, extract it to a method. I think you can create a Base class to hold the permission-checking logic and reuse the created/extracted methods in all the `MVCCommand`. What do you think?